### PR TITLE
Main language in Repository view. Close #119

### LINKED ISSUE #119: Display repository language
The repository's primary programming language string can be found at `repository.language`.

### PR DESCRIPTION
**Pull request by Antoine38660**
Finally positioned top-centered 😄  
Android and iOS preview:

 screenshot_1500504737 
----
simulator screen shot 19 juil 2017 a 17 50 42

### CODE REVIEW

File: src/components/repository-profile.component.js [hunk 1]
```diff
@@ -79,10 +79,33 @@ const styles = StyleSheet.create({
...
+    color: colors.white,
+    paddingLeft: 3,
+    fontSize: normalize(10),
+    ...fonts.fontPrimary,
+  },
 });
 
 export const RepositoryProfile = ({ repository, starred, navigation }: Props) =>
   <View style={styles.container}>
+    {repository.language !== null &&
```
Comment (andrewda) [inline src/components/repository-profile.component.js]: Have you tried this on a repository without a language? I'm pretty sure GitHub responds with an empty string and not `null`, but we may be doing all that conversion somewhere else.

Comment (andrewda) [inline src/components/repository-profile.component.js]: Have you tried this on a repository without a language? I'm pretty sure GitHub responds with an empty string and not null, but we may be doing all that conversion somewhere else.

Comment (Antoine38660) [inline src/components/repository-profile.component.js]: I haven't tried with an empty repo (_without a language_). But I was inspired by https://github.com/gitpoint/git-point/blob/master/src/components/repository-list-item.component.js#L86 (Moreover, the condition in this file is only for the icon node).

Comment (Antoine38660) [inline src/components/repository-profile.component.js]: From the doc (https://developer.github.com/v3/repos/#get) language can be `null` :)

Comment (housseindjirdeh) [inline src/components/repository-profile.component.js]: Yep they return null (I had also assumed they would be empty string but that doesn't seem to be the case). Here's an example of an empty repo (no language) 
https://api.github.com/repos/housseindjirdeh/test-repo

**Pull request merged**